civis/tests/mocks.py

```diff
@@ -37,6 +38,58 @@ def create_client_mock(cache=TEST_SPEC):
     return mock_client
 
 
+def create_client_container_mock(script_id=-10, run_id=100, state='succeeded',
```

Comment:
I feel like something like `create_client_mock_for_container_tests` might be a better name here. With this name, it wasn't clear to me that this would return a mocked API client object.  I don't feel strongly about this.

Reply:
It's a little longer, but I think it is clearer. I'll update.

Reply:
I'm a bit concerned that this new function is too special-purpose to go here. Maybe it should be marked private, and kept for inside this library only? Or else the docstring should get more detail about the structure of the returned mock. As it is, I don't think the function is useful in someone else's tests without them first reading the source code.

Reply:
It's true that it is a more specialized mock, but we need some of these outputs for any testing of a ContainerFuture, so I'm using it across a few test modules. I can add some detail about the structure though, that's a good idea.